frontend/src/components/pages/StoriesPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { gql, useQuery } from "@apollo/client";
+import StoryCard, { StoryCardProps } from "../story/StoryCard";
+
+const GET_STORIES = gql`
+  query GetStories {
+    stories {
+      id
+      title
+      description
+      youtubeLink
+      level
+    }
+  }
+`;
+
+const StoryList = () => {
```

Comment:
Can you make StoryList its own component in `/story` and pass in `stories` as prop? Let's make it as stateless as possible. 

And, can you change this file & route from `StoriesPage` to `Homepage`? ;)

Reply:
Sounds good - what's the benefit of making `StoryList` more stateless?